### ISSUE TITLE
The program keep running for ever when click on Synthesize

### ISSUE THREAD

**Issue by Maenod** [SWivid/F5-TTS#383]
Starting app...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
C:\ProgramData\miniconda3\envs\f5\lib\site-packages\transformers\models\whisper\generation_whisper.py:509: FutureWarning: The input name `inputs` is deprecated. Please make sure to use `input_features` instead.
  warnings.warn(
You have passed task=transcribe, but also have set `forced_decoder_ids` to [[1, None], [2, 50360]] which creates a conflict. `forced_decoder_ids` will be ignored in favor of task=transcribe.
C:\ProgramData\miniconda3\envs\f5\lib\site-packages\transformers\models\whisper\modeling_whisper.py:545: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  attn_output = torch.nn.functional.scaled_dot_product_attention(
Passing a tuple of `past_key_values` is deprecated and will be removed in Transformers v4.43.0. You should pass an instance of `EncoderDecoderCache` instead, e.g. `past_key_values=EncoderDecoderCache.from_legacy_cache(past_key_values)`.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
gen_text 0 My name is mean
Building prefix dict from the default dictionary ...
Loading model from cache C:\Users\maena\AppData\Local\Temp\jieba.cache
Loading model cost 0.940 seconds.
Prefix dict has been built successfully.

**Comment by SWivid** [comment]
@Maenod Hi~ feel free to check the new issue template, which will help better addressing the issue. [here](https://github.com/SWivid/F5-TTS/issues/new?assignees=&labels=help+wanted&projects=&template=help_wanted.yml)
